### PR TITLE
added redhat ga repo

### PR DESCRIPTION
**Pull request by jasinner**

### CODE REVIEW

**Comment by jasinner** [comment]
This change adds the Red Hat GA Maven Repository to the list of repositories searchable in Victims Web. This will allow us to submit victims reports for Red Hat specific versions of artifacts.

File: requirements.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ Flask-Admin>=1.0.7
 Flask-Bcrypt
 Flask-Bootstrap
 Flask-Cache
-Flask-Login>=0.1.1
+Flask-Login==0.2.11
```
Comment (ashcrow) [inline requirements.txt]: Did you test with 0.2.x? Just want to make sure before hand :smile:

**Comment by ashcrow** [comment]
One question. Otherwise looks good!

File: requirements.txt (hunk 1 above)
Comment (jasinner) [inline requirements.txt]: Yes, I tested the app, but did not thoroughly check the unit tests. If we use 0.3 or later of Flask Login we run into a breaking change: https://github.com/maxcountryman/flask-login/blob/master/CHANGES#L25-L26

**Comment by jasinner** [comment]
@ashcrow bump. Anything holding this up?

**Comment by jasinner** [comment]
Thanks for pulling this change @ashcrow. Did you pull the change into the production codebase? I got an error like this when trying to submit a GAV via the web ui: https://github.com/victims/victims-web/issues/130

**Pull request merged**